nni/runtime/msg_dispatcher_base.py

```diff
@@ -46,8 +46,8 @@ def __init__(self, command_channel_url=None):
             self._channel.connect()
         self.default_command_queue = Queue()
         self.assessor_command_queue = Queue()
-        self.default_worker = threading.Thread(target=self.command_queue_worker, args=(self.default_command_queue,))
-        self.assessor_worker = threading.Thread(target=self.command_queue_worker, args=(self.assessor_command_queue,))
+        self.default_worker = threading.Thread(target=self.command_queue_worker, args=(self.default_command_queue,), daemon=True)
```

Comment:
Please add comments to explain why daemon=True, and what problem it fixes.

Reply:
added